pkg/config/config.go

```diff
@@ -85,6 +85,7 @@ func init() {
 	BindEnvAndSetDefault("skip_ssl_validation", false)
 	Datadog.SetDefault("hostname", "")
 	Datadog.SetDefault("tags", []string{})
+	BindEnvAndSetDefault("split_tags", map[string]string{})
```

Comment:
As we have lots of options in datadog.yaml, I like their names to be as explicit as possible. What do you think about `tag_value_split_separator` instead?

Reply:
Personally, I find this to be less explicit: the longer the variable name the harder it is for people to get what it means. I think providing a proper example with a little bit of doc (as in the config template) is worth a thousand word. Maybe there's improvement left to be done on that side of the PR?